MEP_Engine/Query/Geometry.cs

```diff
@@ -39,7 +39,7 @@ public static partial class Query
         
         public static IGeometry Geometry(this IFlow flowObj)
         {
-            return new Line { Start = flowObj.StartPoint, End = flowObj.EndPoint};
+            return new Line { Start = flowObj?.StartPoint, End = flowObj?.EndPoint};
```

Comment:
I returning a line with `null` endpoints safe? Wouldn't it be better to simply return `null` if any of the endpoints of the `IFlow` is `null`?

Reply:
I am fine either way. I implemented your change though